### PR TITLE
Added "/Ping" Command

### PR DESCRIPTION
**Pull request by emericask8ur**
/Ping Command
Signed off by: emericask8ur sk8tj@yahoo.com

### CODE REVIEW

**Comment by windy1** [comment]
Could you please adhere to our code conventions found in our README? Things that need to be changed is to replace all spaces with tabs, include your sign off in the commit message and include a blank line after a code block. It might also be useful to actually get a players ping via the players controller. Thanks :)

**Comment by emericask8ur** [comment]
Added a new pull request for it. Closing this one.

File: src/main/java/org/spout/vanilla/command/AdministrationCommands.java [hunk 1]
```diff
@@ -282,4 +283,9 @@ public void debug(CommandContext args, CommandSource source) throws CommandExcep
 			source.sendMessage("Chunk resent");
 		}
 	}
+        @Command(aliases= "ping", usage = "/Ping", desc = "Check Ping")
+        public void ping(CommandContext args, CommandSource source) throws CommandException {
+            Player player = (Player)source;
+            player.sendMessage("Ping!");      
+     }
```
Comment (Raphfrk) [inline src/main/java/org/spout/vanilla/command/AdministrationCommands.java:290]: This is just to check you are connected to the server ?

Comment (ZNickq) [inline src/main/java/org/spout/vanilla/command/AdministrationCommands.java:290]: In a way, it's a lag test! You can check if your ping is high :)

Comment (windy1) [inline src/main/java/org/spout/vanilla/command/AdministrationCommands.java:290]: Would have been nice if he actually used my getPing method in VanillaPlayer... :p

Comment (ZNickq) [inline src/main/java/org/spout/vanilla/command/AdministrationCommands.java:290]: Meh, that returns a number, most players aren't gonna understand that :/

Comment (windy1) [inline src/main/java/org/spout/vanilla/command/AdministrationCommands.java:290]: /me shrugs

**Pull request closed without merging**